notebooks/evaluation/baseline_evaluation.ipynb

```diff
@@ -0,0 +1,183 @@
val?', '', '']), (401, ['Describe MAP with gaussian prior.', '', '']), (402, ['Describe the setup of a Markov Decision Process.', 'What does the setup of a Markov Decision Process include?', '']), (403, ['How is an information retrieval model formally defined? ', '', '']), (404, ['What is an example for a Heap Overflow Attack?', '', '']), (405, [\"Explain the lloyd's algorithm for K means.\", '', '']), (406, ['How to compute the robustness of protocols for replication transparency?', '', '']), (407, ['What is a tiered index?', 'How to look-up documents with a tiered index?', '']), (408, ['How are tests and test paths denoted and what does syntactic/semantic reach mean?', 'What is syntactic/semantic reach?', 'How are tests and test paths denoted?']), (409, ['What is a problem of the rainbow table approach and how is it solved?', '', '']), (410, ['What is an example for a Race Condition Attack? Name and explain how it works', '', '']), (411, ['How is the expected discounted reward value of following policy pi in state s mutually recursively defined?', 'How is the total expected reward value of doing a in state s, then following policy pi mutually recursively defined?', '']), (412, ['When does the exchange phase become difficult?', '', '']), (413, ['What are determining predicates?', '', '']), (414, ['What are the problems of knowledge integration?', '', '']), (415, ['What is Tor?', '', '']), (416, ['Explain Structural Properties.', '', '']), (417, ['How are wordy grouped in a MapReduce program?', '', '']), (418, ['What are the implications of the work of Lebovitz et al. (2022)?', '', '']), (419, ['How is externally regulated behavior defined and how is it related to intrinsically motivated behavior?', 'What is Internalization?', '']), (420, ['How does locality sensitive hashing work?', '', '']), (421, ['How are infeasible test requirements handled in logic coverage criteria?', '', '']), (422, ['How is Base Choice Coverage defined?', '', '']), (423, ['What is an example for an SQL Injection Attack?', '', '']), (424, ['What is the concept of Critical Data?', '', '']), (425, ['What is a click fraud attack?', '', '']), (426, ['What is leadership according to famous personalities or movies?', '', '']), (427, ['Explain geometric interpretation of SVD.', '', '']), (428, ['Explain entropy.', 'What is Shannon entropy?', '']), (429, ['Which differences exist between genders (men and women) in leadership effectiveness?', '', '']), (430, ['What is Bitcoin and what is its goal and advantage?', '', '']), (431, ['What are the main differences between explicit and tacit knowledge?', '', '']), (432, ['What are general advices regarding potentially harmful e-mails?', '', '']), (433, ['How to recover when an agent has crashed?', '', '']), (434, ['How does the merge algorithm work?', '', '']), (435, ['How is the relevance score computed if relevance judgements are available?', '', '']), (436, ['What is the relationship between Bernoulli model and odds?', '', ''])]\n",
```

Comment:
Please use `df.head(5)` to print only a few lines and check if everything worked. There is no need to print everything here. This makes the notebook very unreadable

Reply:
Cleared the output. Complete table was needed to scan through all results to find wrong labeled rows